### PR TITLE
Feature/rstudio

### PR DESCRIPTION
**Pull request by sandstromviktor**
Building Rstudio-server from scratch on an ubuntu 20.04 image. It sets up it correctly and starts the server with a user called `rstudio-server` which has no root access. 
`python docker`  is used to start the container where `pytest` tests the container is different ways.

### CODE REVIEW

**Comment by sandstromviktor** [comment]
Did too much in this branch. I added trivy tests to TorchServe aswell. This was a mistake and i can remove it if you want @alfredeen?

**Comment by alfredeen** [comment]
> Did too much in this branch. I added trivy tests to TorchServe aswell. This was a mistake and i can remove it if you want @alfredeen?

I will leave that up to you but sounds like a good idea.

File: rstudio/tests/test_connection.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+def test_rstudio_port():
+    assert container.ports["8787/tcp"][0]["HostPort"] == "8787"
+
+
+def test_rstudio_access():
+    rstudio_url = "http://{}:8787".format(
+        container.attrs["NetworkSettings"]["Networks"]["bridge"]["IPAddress"]
+    )
+    response = requests.get(rstudio_url)
```
Comment (alfredeen) [inline rstudio/tests/test_connection.py:27]: Recommended to set a timeout on the GET request.

**Comment by alfredeen** [comment]
Great work

**Comment by sandstromviktor** [comment]
Needs to be merged.

**Comment by alfredeen** [comment]
Great

**Review by alfredeen — APPROVED** [review]
Great work

**Pull request merged**